### ISSUE TITLE
Add eth_createAccessList to Geth docs

### ISSUE THREAD

**Issue by snario** [ethereum/go-ethereum#23588]
It is currently difficult to see if this API is implemented at all and what the API actually is from the docs, for which there are none. It should be here I think(?): https://geth.ethereum.org/docs/rpc/ns-eth

**Comment by MariusVanDerWijden** [comment]
Yep this should be part of the spec's if someone wants to take over, this is a great beginner task

**Comment by verheesj** [comment]
Please assign this to me :)
